payment-in-stablecoins/README.md

```diff
@@ -0,0 +1,95 @@
+# Proposal Payment in Stablecoins
+
+## Simple Summary
+
+Allows the DAO to execute proposals that send funds denominated in USD. This change addresses the ETH/USD price volatility risk both sides suffer from today, as well as the DAO's ability to perform ETH-to-USD swaps trustlessly.
+
+## Abstract
+
+This specification introduces two new contracts that facilitate stablecoin payments:
+
+- `NOU` a new ERC20 token the DAO can mint to say "I owe you this amount of USD"
+- `NounsStablecoinPayments` a contract that:
+  - allows the DAO to mint NOU tokens as payment to proposal recipients
+  - allows anyone to sell it stablecoins in exchange for ETH with some additional incentive
+  - allows proposal recipients to redeem NOU for stablecoins
+
+## Technical Specification
+
+### Overview
```

Comment:
let's add the contract interactions step by step:

1. builder creates proposal asking for $10K:
    1. Assuming ETH/USD = 1700
    2. includes a transaction:
    `NounsStablecoinPayments.mintNOU(builderWallet, 10_000)` with ETH value of `NounsStablecoinPayments.ethNeeded(10_000, 2)`
2. Proposal executes:
    1. builderWallet is minted 10K NOU tokens
    2. NounsStablecoinPayments is funded with ~11.7 ETH
3. Arbitrageurs buy ETH in exchange for USD by calling `NounsStablecoinPayments.sellStablecoin(10_000)`
    1. This will happen when the contract price is more favorable than current market price
    2. `NounsStablecoinPayments` has a 10K USD balance now
4. Builder redeems USD by calling `NounsStablecoinPayments.redeem()`
    1. Builder's NOU tokens are burned
    2. `NounsStablecoinPayments` transfers 10K USD to builder

Reply:
agreed, will add